src/legacy/ui/public/new_platform/new_platform.ts

```diff
@@ -74,7 +74,7 @@ export async function __newPlatformStart__(
 }
 
 export function getNewPlatform() {
-  if (runtimeContext.setup.core === null || runtimeContext.start.core === null) {
+  if (runtimeContext.setup.core === null /* || runtimeContext.start.core === null */) {
```

Comment:
My understanding was that platform loading follows

```
NP setup -> NP start -> legacy platform
```

but for some reason when legacy runs `runtimeContext.start.core` is still null.

Reply:
Does this need to be resolved before merging?

Reply:
We don't need to merge this PR, I should have made it as a Draft. I guess, this is just an example of moving one Interpreter registry, but I am looking into moving way bigger chunk of Interpreter, as it does not have many dependencies.

But in general this `runtimeContext.start.core` must be addressed somehow. For moving registries we don't need `runtimeContext.start.core`, but `kibana-react` will probably need `runtimeContext.start.core` in legacy world.